generators/app/tasks/appSetup/drawerFeatureFiles.js

```diff
@@ -1,37 +1,46 @@
+const DRAWER_FEATURE_PATH = ['src', 'app', 'components', 'Drawer'];
+const DRAWER_MENU_FEATURE_PATH = ['src', 'app', 'components', 'Drawer', 'components', 'DrawerMenu'];
+const DRAWER_OVERLAY_FEATURE_PATH = ['src', 'app', 'components', 'Drawer', 'components', 'DrawerOverlay'];
+
 module.exports = function drawerFeatureFiles() {
-  // src/components/Drawer/DrawerContainer.js
+  // src/app/components/Drawer/index.js
   this.fs.copy(
-    this.templatePath('src', 'components', 'Drawer', 'DrawerContainer.js'),
-    this.destinationPath(this.projectName, 'src', 'components', 'Drawer', 'DrawerContainer.js')
+    this.templatePath(...DRAWER_FEATURE_PATH, 'index.js'),
+    this.destinationPath(this.projectName, ...DRAWER_FEATURE_PATH, 'index.js')
   );
-  // src/components/Drawer/DrawerMenu.js
+  // src/app/components/Drawer/components/DrawerMenu/index.js
   this.fs.copy(
-    this.templatePath('src', 'components', 'Drawer', 'DrawerMenu.js'),
-    this.destinationPath(this.projectName, 'src', 'components', 'Drawer', 'DrawerMenu.js')
+    this.templatePath(...DRAWER_MENU_FEATURE_PATH, 'index.js'),
+    this.destinationPath(this.projectName, ...DRAWER_MENU_FEATURE_PATH, 'index.js')
   );
-  // src/components/Drawer/DrawerMenu.styles.js
+  // src/app/components/Drawer/components/DrawerMenu/styles.js
   this.fs.copy(
-    this.templatePath('src', 'components', 'Drawer', 'DrawerMenu.styles.js'),
-    this.destinationPath(this.projectName, 'src', 'components', 'Drawer', 'DrawerMenu.styles.js')
+    this.templatePath(...DRAWER_MENU_FEATURE_PATH, 'styles.js'),
+    this.destinationPath(this.projectName, ...DRAWER_MENU_FEATURE_PATH, 'styles.js')
   );
-  // src/components/Drawer/DrawerMenuContainer.js
+  // src/app/components/Drawer/components/DrawerMenu/layout.js
   this.fs.copy(
-    this.templatePath('src', 'components', 'Drawer', 'DrawerMenuContainer.js'),
-    this.destinationPath(this.projectName, 'src', 'components', 'Drawer', 'DrawerMenuContainer.js')
+    this.templatePath(...DRAWER_MENU_FEATURE_PATH, 'layout.js'),
```

Comment:
@IgnacioAbadie Notice that we are copying the container first and then the layout. What we can see here is a reorder, but nothing strange; since there is a permutation of these `fs.copy`. The result here should be the same.